app/services/github_handler.rb

```diff
@@ -45,6 +45,10 @@ def handle_review_request
     pr.review_requests.create!(data: payload, owner: owner, reviewer: reviewer)
   end
 
+  def handle_review_removal
+    PullRequest.find_by!(github_id: payload['pull_request']['id']).destroy!
```

Comment:
Maybe I'm missing something, but why removing a review request destroys a `PullRequest`? I thought we would be just destroying the `ReviewRequest`

Reply:
Yeah, i got a miss understanding. Thanks!